jalib/jassert.cpp

```diff
@@ -302,7 +288,8 @@ void jassert_internal::open_log_file()
   if (theLogFileFd != -1) {
     dmtcp::ostringstream a;
 
-    a << "[" << getpid() << "] INFO at " << JASSERT_FILE << ":" << JASSERT_LINE
+    a << "[" << jalib::getTimestampStr() << ", " << getpid() << ", "
```

Comment:
@karya0, should we also add the keyword PID(or P) or TID while logging? It would be clear to anyone who's not familiar with the logging format.

Reply:
Putting Pid/Tid in log lines is a fairly standard practice so we don't have to worry about people unfamiliar with it :).